### ISSUE TITLE
accessibility: adapt-contrib-media mejs code not good

### ISSUE THREAD

**Issue by oliverfoster** [adaptlearning/adapt_framework#2190]
### Subject of the issue
* General key trap listeners block screen reader navigation
* Default seek by keypress behaviour is broke
* Broken forms mode with `role="application"` and  `aria-controls='mep_'` combination
* Video tag gets screen reader focus on play
* Media player width pushes ios iframe to scale

### Expected behaviour
Should be able to walk into the player and be able to use the play button, time slider, mute and fullscreen buttons

**Comment by oliverfoster** [comment]
merged
